core/views.py

```diff
@@ -1 +1,39 @@
-# Create your views here.
+from django.shortcuts import get_object_or_404
+from django.views.generic import ListView, TemplateView
+
+from .models import Post
+
+
+class PostListlView(ListView):
+
+    """Provide a temporary view for internal testing of the post view count feature.
+
+    This view is not intended for production use and should be accessed only by developers.
+    """
+
+    model = Post
+    template_name = "post-list.html"
+    context_object_name = "posts"
+
+
+class PostDetailView(TemplateView):
+
+    """Provide a temporary view for internal testing of the post view count feature.
+
+    This view is not intended for production use and should be accessed only by developers.
+    """
+
+    template_name = "post-detail.html"
+
+    def get_context_data(self: "PostDetailView", **kwargs: dict) -> dict:
+        context = super().get_context_data(**kwargs)
+
+        post_id = self.kwargs.get("pk")
+        post = get_object_or_404(Post, pk=post_id)
+        post.view_count += 1
+        post.save(update_fields=["view_count"], skip_version_check=True)
```

Comment:
I am thinking it should be update as here: 
https://github.com/pymasterspl/reddit/blob/fe257582955c20ea67e95b781a348c43d0d6785e/users/models.py#L53-L54
Reasoning: whenever save is run, all signals are run - post saves and all this that django runs. We don't need that, we only want to bump +1 to the read count. 
So please change to update.

Reply:
You can try to use F() expression that is great for db performance
[https://docs.djangoproject.com/en/5.0/ref/models/expressions/#f-expressions](url)